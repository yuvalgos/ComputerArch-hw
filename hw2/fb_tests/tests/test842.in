w 0x7eae4
r 0x7eb10
r 0x7eafc
r 0x7eaec
r 0x7eaf4
r 0x7eb0c
r 0x7eafc
r 0x7eb24
r 0x7eaf0
w 0x7eb0c
w 0x7eac4
r 0x7eb24
w 0x7eb08
r 0x7eaf8
w 0x7eb20
r 0x7eb2c
w 0x7eab4
r 0x7eabc
r 0x7eadc
r 0x7eac0
r 0x7ead8
r 0x7eb40
w 0x7eb1c
r 0x7eaac
w 0x7eac4
w 0x7eb40
w 0x7eaac
w 0x7eb24
w 0x7eac8
w 0x7eac4
r 0x7eb04
w 0x7eabc
r 0x7eb20
r 0x7eacc
w 0x7eaf8
w 0x7eabc
r 0x7eb00
w 0x7eb20
w 0x7eab4
w 0x7eab8
w 0x7eae0
