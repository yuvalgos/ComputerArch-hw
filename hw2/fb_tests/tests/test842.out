L1miss=0.512 L2miss=1.000 AccTimeAvg=120.902
